enarx-keep-sgx-shim/src/handler.rs

```diff
@@ -641,6 +641,33 @@ impl<'a> Handler<'a> {
 
         trusted.len() as u64
     }
+
+    // Do clock_gettime syscall
+    pub fn clock_gettime(&mut self) -> u64 {
+        self.trace("clock_gettime", 2);
+
+        let clk_id = self.aex.gpr.rdi.raw();
+        let ts = self.aex.gpr.rsi.raw() as *mut libc::timespec;
+
+        unsafe {
+            let uaddr: *mut libc::timespec = core::mem::transmute(&mut self.block.buf[0]);
```

Comment:
Don't use `transmute` here. We don't know that the alignment of the buffer is correct. Instead, use https://doc.rust-lang.org/std/primitive.slice.html#method.align_to in order to get a reference to a `timespec`. If you do this, much more of this code will be safe.